src/components/form/book-submit.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+class Form extends React.PureComponent {
+  render() {
+    return (
+      <>
+        <h2>Add New Book</h2>
+        <form action="#">
+          <div className="form-group">
+            <input
+              type="text"
+              className="form-control"
+              id="title"
+              placeholder="Title"
+            />
+          </div>
+          <div className="form-group">
+            <input
+              type="text"
+              className="form-control"
+              id="author"
+              placeholder="Author"
+            />
+          </div>
+          <button type="submit" className="btn btn-primary">
+            Add Book
+          </button>
+        </form>
```

Comment:
✅ Form component renders a form with inputs for the title and the author of the book 👍🏻 
✅ Has a button to submit 👍🏻